File: THCS/2023_2024/DakNong_2324/Bai1/TICKET.py
def main():
    # Đọc dữ liệu từ file TICKET.INP
    with open("TICKET.INP", "r") as input_file:
        a, b, u, v, x, y = map(int, input_file.readline().strip().split())

    # Tính số người lớn
    adults = x - y

    # Tính tổng chi phí
    total_cost = (a * adults) + (b * y) + (u * adults) + (v * y)

    # Ghi kết quả vào file TICKET.OUT
    with open("TICKET.OUT", "w") as output_file:
        output_file.write(str(total_cost))

if __name__ == "__main__":
    main()

File: THCS/2023_2024/DakNong_2324/Bai1/TICKET.cpp
#include <iostream>
#include <fstream>

int main() {
    // Khai báo các biến
    int a, b, u, v, x, y;
    
    // Đọc dữ liệu từ file TICKET.INP
    std::ifstream input("TICKET.INP");
    input >> a >> b >> u >> v >> x >> y;
    input.close();

    // Tính số người lớn
    int adults = x - y;

    // Tính tổng chi phí
    int total_cost = (a * adults) + (b * y) + (u * adults) + (v * y);

    // Ghi kết quả vào file TICKET.OUT
    std::ofstream output("TICKET.OUT");
    output << total_cost;
    output.close();

    return 0;
}